### PR TITLE
treesitter: add ignoreInstall option to treesitter config

### PR DESCRIPTION
**Pull request by ForsakenHarmony**
Necessary for me because of https://github.com/claytonrcarter/tree-sitter-phpdoc/issues/15

### CODE REVIEW

File: plugins/languages/treesitter.nix [hunk 1]
```diff
@@ -15,9 +15,15 @@ in
         description = "Install grammars with Nix (beta)";
       };
       ensureInstalled = mkOption {
-        type = with types; oneOf [ (enum [ "all" "maintained" ]) (listOf str) ];
-        default = "maintained";
```
Comment (ForsakenHarmony) [inline plugins/languages/treesitter.nix:19]: `maintained` also seems like it no longer exists?

Comment (pta2002) [reply plugins/languages/treesitter.nix:19]: Yep, it no longer exists! Haven't updated this module in a while though, should get rid of "maintained" and default to "all"

Comment (ForsakenHarmony) [reply plugins/languages/treesitter.nix:19]: Do you not use `treesitter`? what's your alternative?

Comment (pta2002) [reply plugins/languages/treesitter.nix:19]: Oh I do, just had it set to all in my personal config and didn't notice the change

**Pull request merged**